src/App.css

```
@@ -1744,8 +1878,8 @@ body {
 }
 
 .entity-token.is-ok {
-  --chip-status-color: #05d5fa;
-  box-shadow: 0 0 8px rgba(5, 213, 250, 0.12);
+  --chip-status-color: #10b981;
+  box-shadow: 0 0 8px rgba(16, 185, 129, 0.12);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)